application/helpers/update/updates/Update_607.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use Exception;
+
+/**
+ * Fix organizer link : icon and survey activated
+ * @package LimeSurvey\Helpers\Update
+ */
+class Update_607 extends DatabaseUpdateBase
+{
+    /**
+     * @inheritDoc
+     * @throws \CException
+     */
+    public function up()
+    {
+        $surveyThemes = $this->db->createCommand()
+            ->select('*')
+            ->from('{{templates}}')
+            ->queryAll();
+        if (!empty($surveyThemes)) {
```

Comment:
If you want to avoid excessive indentation, you can do an early return here instead

```php
if empty $surveyThemes
  return;
```